ClearKeep/Scenes/CreateDirectMessage/CreateDirectMessageView.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  CreateDirectMessageView.swift
+//  ClearKeep
+//
+//  Created by MinhDev on 01/04/2022.
+//
+
+import SwiftUI
+import Common
+import CommonUI
+
+struct CreateDirectMessageView: View {
+	// MARK: - Constants
+	private let inspection = ViewInspector<Self>()
+
+	// MARK: - Variables
+	@Environment(\.colorScheme) var colorScheme
+	@Environment(\.injected) private var injected: DIContainer
+	@State private(set) var imageUser: Image
+	@State private(set) var userName: String
+	@State private(set) var message: String
+
+	// MARK: - Init
+	init(imageUser: Image,
+		 userName: String = "",
+		 message: String = "") {
+		self._imageUser = .init(initialValue: imageUser)
+		self._userName = .init(initialValue: userName)
+		self._message = .init(initialValue: message)
+	}
+
+	// MARK: - Body
+	var body: some View {
+		content
+			.onReceive(inspection.notice) { inspection.visit(self, $0) }
+			.navigationBarTitle("")
+			.navigationBarHidden(true)
+			.edgesIgnoringSafeArea(.all)
+			.background(backgroundColorView)
+	}
+}
+
+// MARK: - Private
+private extension CreateDirectMessageView {
+	var content: AnyView {
+		AnyView(notRequestedView)
+	}
+}
+
+// MARK: - Private Variables
+private extension CreateDirectMessageView {
+	var backgroundColorView: Color {
+		colorScheme == .light ? AppTheme.shared.colorSet.background : AppTheme.shared.colorSet.black
+	}
+}
+
+// MARK: - Loading Content
+private extension CreateDirectMessageView {
+	var notRequestedView: some View {
+		DirectMessageContentView(imageUser: $imageUser, userName: $userName, searchText: .constant(""), severText: .constant(""), inputStyle: .constant(.default))
```

Comment:
@Minhdn1 : If not required, please set default value at initial function

Reply:
fix complete